Sources/SwiftTUI/Drawing/CGPoint+Position.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+let pixelBlockSize = (width: 4.0, height: 8.0)
```

Comment:
Braille pattern consists of a 2x4 dot matrix, but we used a pixel block size of 4x8 (as in `Image`, #5), which should more closely match the typical aspect ratio of a monospace font.

Reply:
It might be useful to use a `scale` parameter when converting between `CGPoint`, `CGSize`, `CGRect` and `Position`, `Size`, and `Rect`.

Reply:
In #8, `pixelBlockSize` is a property of the renderer.